docs/en/compute/Hive.md

```diff
@@ -72,7 +76,7 @@ creates:
 
 Then create a new internal table:
 
-```
+```sql
```

Comment:
another enforcement type, all code blocks that start with `hive>` should be a sql code block. cc @Zijian-Zhu